lldb/test/Shell/helper/toolchain.py

```diff
@@ -165,6 +235,14 @@ def use_support_substitutions(config):
     if config.cmake_sysroot:
         host_flags += ["--sysroot={}".format(config.cmake_sysroot)]
 
+    if config.enable_remote and config.has_libcxx:
+        host_flags += [
+            "-L{}".format(config.libcxx_libs_dir),
+            "-Wl,-rpath,{}".format(config.libcxx_libs_dir),
+            "-lc++",
+            "-lc++abi",
```

Comment:
These flags don't seem very consistent. -rpath is only useful for dynamic links but I think you're using static links now. And since we removed -lc++abi (which is only needed for *some* static links) from the makefile, I don't think it should be used here as well.

Reply:
Fixed.